Additions/getAddition.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BasicOperations;
+
+namespace Additions
+{
+        public class getAddition : ISum
+        {
+            public dynamic result;
+            public dynamic Add(dynamic a, dynamic b)
+            {
+                result = Addition.Sum(a, b);
+                return result;
+            }
+            public dynamic Add(dynamic a)
+            {
+                result = Addition.Sum(a);
+                return result;
+            }
```

Comment:
The call to this is what is missing in the interface. Please include it before merging